hr_time/templates/worklog/worklog_textbox.html

```diff
@@ -0,0 +1,15 @@
+<div class="d-flex justify-between" id="worklog_template">
+    <label id="worklog_section_label" class="control-label">{{ _("Add Worklog") }}</label>
+    <button class="btn btn-xs btn-outline-info edit-full-form-btn" type="button">
+        {{ _("Enter complete detail") }}
+        &nearr;
+    </button>
```

Comment:
If possible, please try to move the full form button to the end of the textbox.
This looks better from alignment point of view and fixes also some rendering issues on mobile devices.

Expected result:
![Bildschirmfoto vom 2024-11-06 13-37-33](https://github.com/user-attachments/assets/13759ae5-d164-43b1-9045-ee6c7df8bba1)

Reply:
Understood.
But could you explain the reason for the round blue check mark icon ?
When should it be displayed / hidden ?